display_blocks.js

```diff
@@ -161,10 +157,116 @@ function parseObject(obj) {
     var nodeInstPVList = document.createElement("ul");
 
     nodeInstPVs.appendChild(nodeInstPVList);
+    nodeInstPVs.appendChild(nodeInstPVList);
```

Comment:
Why is this line needed twice? Please add comment to explain why it's needed (or remove if not needed).